YouTube.js

```diff
@@ -90,10 +90,10 @@ function doWeb(doc, url) {
 function scrape(doc, url) {
 	var item = new Zotero.Item("videoRecording");
 	item.title = text(doc, '#info-contents h1.title');
-	item.url = url;
+	item.url = attr(doc, 'link[rel="canonical"]', 'href') || url;
 	item.runningTime = text(doc, '#movie_player .ytp-time-duration');
-	
-	item.date = text(doc, '#info-text #date');
+
+	item.date = attr(doc, 'meta[itemprop="datePublished"]', 'content');
```

Comment:
Oh God. I remember now why we don't use the meta tags for YouTube. Try going to a video, clicking on a link to a related video (so that it opens in the same tab), then opening browser developer tools and looking at the meta/link tags. They don't change! Not even `<link rel="canonical">`. They reflect the metadata of the original video, not the new one you opened. So we just can't use them.

Could we stick with screen scraping but just update the selector? I think it should be:

```js
item.date = ZU.strToISO(text(doc, '#info-strings yt-formatted-string'));
```